src/Package/Fetch.zig

```diff
@@ -616,11 +647,13 @@ fn loadManifest(f: *Fetch, pkg_root: Cache.Path) RunError!void {
 
     f.manifest = try Manifest.parse(arena, ast.*, .{
         .allow_missing_paths_field = f.allow_missing_paths_field,
+        .allow_missing_id = f.allow_missing_paths_field,
+        .allow_name_string = f.allow_missing_paths_field,
```

Comment:
I tried building the branch and running `zig fetch` to re-fetch and re-hash packages and it failed with an "expected enum literal" error.

`cmdFetch` sets `.allow_missing_paths_field = false`, so if the intention is for users to be able to fetch legacy packages until they are wholly deprecated, you might want to break this out into a separate field.

Reply:
Good find; fixed in a commit to be pushed shortly.